Build fails

Trying to build fails during Step 22 (RUN ./configure).  I get the following:

---

debconf: delaying package configuration, since apt-utils is not installed
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (This frontend requires a controlling tty.)
debconf: falling back to frontend: Teletype

---

invoke-rc.d: policy-rc.d denied execution of restart.

---
checking for Expat XML Parser... no
configure: error: required library not found
configure: error: ./configure failed for osm2pgsql
The command '/bin/sh -c ./configure' returned a non-zero code: 1
(Quits out right here)

---

I get similar errors when trying to build other iterations of nominatim as well, however they are all slightly different, though all during the ./configure step.

Any thoughts on why this might be happening and how to resolve it?

Thanks!

--- a/Dockerfile
+++ b/Dockerfile
@@ -12,6 +12,7 @@ RUN apt-get -y install build-essential
 RUN apt-get -y install gcc git osmosis
 RUN apt-get -y install libxml2-dev libgeos-dev libpq-dev libbz2-dev libtool automake libproj-dev
 RUN apt-get -y install proj-bin libgeos-c1 libgeos++-dev
+RUN apt-get -y install libexpat1-dev
 
 # Install Boost (required by osm2pqsql)
 RUN apt-get -y install autoconf make g++ libboost-dev \

